src/win.h

```diff
@@ -464,6 +464,9 @@ bool win_check_flags_all(struct managed_win *w, uint64_t flags);
 /// Mark properties as stale for a window
 void win_set_properties_stale(struct managed_win *w, const xcb_atom_t *prop, int nprops);
 
+/// Determine if a window should animation
+bool attr_pure win_should_animation(session_t *ps, const struct managed_win *w);
```

Comment:
Same things as in the .c file: fix the english in the comment and change the function name.